docs/escape_support.md

```diff
@@ -65,7 +65,10 @@ brevity.
 | `CSI l`    | PARTIAL     | See `CSI h` for supported modes                   |
 | `CSI ? l`  | PARTIAL     | See `CSI ? h` for supported modes                 |
 | `CSI M`    | IMPLEMENTED |                                                   |
-| `CSI m`    | IMPLEMENTED |                                                   |
+| `CSI m`    | PARTIAL     | Supported parameters:                             |
+|            |             |   `0`-`9`, `21`-`25`, `27`-`49`, `58`, `59`       |
+|            |             |   `90`-`97`, `100`-`107`                          |
```

Comment:
I'd just keep this as `IMPLEMENTED` with nothing on the right side, since those are pretty much all the SGR escape sequences that I'm aware of.

Reply:
Those are pretty much all unused and can be considered rejected, yes. Probably not a terrible idea to keep the supported parameters list on the right, but `PARTIAL` I'd still replace with `IMPLEMENTED` since as far as I'm aware we've got pretty much everything in use covered.

Reply:
I must again contest the suggestion that SGR 51-55 are unused, but that seems to be a moot point.

Reply:
They certainly are and have been explicitly rejected in the past: https://github.com/alacritty/alacritty/issues/1647